### PR TITLE
Add example for request params

### LINKED ISSUE #378: Add Example for request params
Example for handling params from request is missing. 

I discovered jsonrpc uses `parse` and method to parse Params struct.

Is parsing request parameters in progress?

### PR DESCRIPTION
**Pull request by hskang9**
resolves #378

### CODE REVIEW

**Comment by hskang9** [comment]
[clabot:check]

File: core/examples/params.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use jsonrpc_core::*;
+
+fn main() {
+	let mut io = IoHandler::new();
+
+	io.add_method("say_hello", |_params: Params| {
+    let parsed: Value = _params.parse().unwrap();
```
Comment (tomusdrw) [inline core/examples/params.rs]: Parsing to `Value` is not super useful, `.parse()` allow you to get anything that can be deserialized:
```
#[derive(Deserialize)]
struct HelloParams {
  name: String,
}
...

let parsed: HelloParams = params.parse().ok_or_else(Error::invalid_params)?;
Ok(Value::String(format!("{}", parsed.name));
```
Would be a more appropriate example imho.

Also:
1. Indentation is off (please reformat the entire file according to our `.editorconfig` rules).
2. `_params` should be renamed to `params` (the `_` in front indicates that parameter is unused)

Comment (hskang9) [reply core/examples/params.rs]: I see. Thank you for the review, and I will update it shortly.

File: core/examples/params.rs [hunk 2]
```diff
@@ -1,15 +1,21 @@
 use jsonrpc_core::*;
+use serde::{Deserialize};
```
Comment (tomusdrw) [inline core/examples/params.rs]: You need `serde_derive` for this.

Comment (hskang9) [reply core/examples/params.rs]: omw

**Comment by tomusdrw** [comment]
Thanks!

**Comment by hskang9** [comment]
No problem!

**Pull request merged**